.circleci/config.yml

```diff
@@ -132,7 +132,6 @@ workflows:
       - test-packages
       - lint
       - build
-      - ci/validate-commits
```

Comment:
So we don't want to validate commit messages anymore?

Reply:
The present tool for validate messages requires jira information, which seems to not align with our documentation for public repos.

Reply:
Ok, I recommend updating `validate-commits` to disallow JIRA references for public repos. Otherwise, if we can't enforce it programatically, we will end up with inconsistent commit messages and not able to achieve our goal.

Reply:
Yes that sounds like a reasonable way forward 👍